### ISSUE TITLE
Radar module crashing.

### ISSUE THREAD

**Issue by c615586** [jojoduquartier/MMM-RAIN-RADAR#12]
The MMM-RAIN-RADAR app has been crashing after about 5 minutes for the last 3-4 days at least.  It returns when I reboot the Magic Mirror, but then crashes again.  

Anyone else have this issue?

**Comment by jawithee** [comment]
I am getting the same issue now. Did you resolve the problem?

**Comment by lhops9331** [comment]
Yes, I have been having the same problem.  MMM Rain-Radar has been working fine for over a month and in the last few days it has started going blank on the screen or crashing.  Reboot brings it back but only for a few minutes. Were you able to get it fixed?
